.travis.yml

```diff
@@ -3,10 +3,17 @@ language: php
 php:
     - 5.5
     - 5.6
+    - 7.0
     - hhvm
 
+matrix:
+    include:
+        - php: 5.5
+          env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
+
 before_script:
-    - travis_retry composer install --no-interaction
+    - travis_retry composer self-update
```

Comment:
As far as I know its a good practice to update composer before running a build, sure it may add a few extra seconds, but it gets you a decent version of composer.